### PR TITLE
JSON errors

### LINKED ISSUE #33: Add error annotations in ace editor
It could be nice to have error annotations in the editor in addition to the error messages we have right now when compiling.

### PR DESCRIPTION
**Pull request by paf31**
This adds a red underline to the text where the error occurred.

Fixes #33.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -553,10 +556,41 @@ $(function() {
...
+                                $('#column2')
+                                    .append($('<h1>').addClass('error-banner').append("Error " + (i + 1) + " of " + errors.length))
+                                    .append($('<pre>').append($('<code>').append(error.message)));
+
+                                // Add an error marker
+                                var range = new (ace.require("ace/range").Range)
+                                                ( error.position.startLine - 1
+                                                , error.position.startColumn - 1
+                                                , error.position.endLine - 1
+                                                , error.position.endColumn - 1);
```
Comment (paf31) [inline js/index.js]: Ah, that's why. Yes, I'll figure something out, thanks.

Comment (paf31) [inline js/index.js]: @soupi This should be fixed now. Well spotted :)

File: js/index.js [hunk 2]
```diff
@@ -338,7 +338,7 @@ $(function() {
                    };
         } else { // core
             return { backend: "core",
-                     endpoint: "https://compile.purescript.org/try",
+                     endpoint: "http://localhost:8080",
```
Comment (soupi) [inline js/index.js]: forgot this :)

**Comment by soupi** [comment]
cool stuff 😃

**Pull request merged**